docs/configuration/runs.md

```diff
@@ -0,0 +1,141 @@
+Run Configurations
+==================
+
+Run configurations define how an instance of the game is going to run. This includes arguments, working directories, task names, etc. Run configurations are defined within the `minecraft.runs` block. While no runs are configured by default, ForgeGradle does provide some assumptions for the configurations `client`, `server`, `data`, or `gameTestServer` on their entry points.
```

Comment:
These configurations are provided by Forge's userdev config (located at `config.json`) and not FG.